### DISCUSSION TITLE
R Icon for repositories

### DISCUSSION THREAD

**Discussion by siduojiang** [community/community#23796] [New to GitHub]
<p>Hi there,</p>
<p>I apologize if this is a basic question but I am having a difficult time finding an answer. </p>
<p>When a respository is created with say a Jupyter Notebook, there’s automatically an orange “Jupyter Notebook” icon that gets added under the repository description. I don’t get this when I add a repository with R code. Is there a way to manually add something similar for Rmd or R files? </p>
<p>Thanks in advance.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! I think you’re talking about the “Language Bar” that every repository has?</p>
<p>GitHub determines language statistics based on the total size of the files that belong to each language.</p>
<p>We use the Linguist open source library to analyze source code files. If your files aren’t showing up with the language you want them to be, Linguist supports a number of different custom overrides strategies for language definitions and vendored paths. These are commonly used for documentation, vendor files, or generated files. You can read about how to use overrides in the Linguist documentation:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/github/linguist#troubleshooting" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181079362-ccd0428f-c644-4376-ba8b-eb72ef1800de.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/github/linguist#troubleshooting" target="_blank">github/linguist</a></h3>

<p>Language Savant. If your repository's language is being reported incorrectly, send us a pull request! - github/linguist</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I just had a look at your GitHub profile, and it looks like many of your repos are correctly showing up with a blue R icon already, so it doesn’t look like you need to adjust those settings.</p>
